c/pthread/stack_false-unreach-call.c

```diff
@@ -38,7 +38,7 @@ int get_top(void)
 
 int stack_empty(void)
 {
-  (top==0) ? TRUE : FALSE; 
+  return (top==0) ? TRUE : FALSE;
```

Comment:
I agree that this is probably the intention of the program, but does this change affect the semantics of the program, i.e., the verification result?

Reply:
In this case, there is no change in the semantics, as the function is never called in the benchmark. If the function was ever called, then, according to the C standard, the program would not be defined as every function with non-void return type (with the exception of main) has to end with return.